cmake/Package.cmake

```
@@ -0,0 +1,23 @@
+# -- packaging -----------------------------------------------------------------
+
+set(CPACK_PACKAGE_VENDOR "Tenzir")
+set(CPACK_PACKAGE_CONTACT "engineering@tenzir.com")
+set(CPACK_VERBATIM_VARIABLES ON)
+
+set(CPACK_RESOURCE_FILE_LICENSE "${CMAKE_CURRENT_SOURCE_DIR}/LICENSE")
+set(CPACK_RESOURCE_FILE_README "${CMAKE_CURRENT_SOURCE_DIR}/README.md")
+set(CPACK_INSTALLED_DIRECTORIES "/var/lib/vast" "/var/log/vast")
+
+set(CPACK_GENERATOR TZST;DEB)
+set(CPACK_DEBIAN_COMPRESSION_TYPE gzip)
+set(CPACK_DEBIAN_FILE_NAME DEB-DEFAULT)
+set(CPACK_DEBIAN_PACKAGE_NAME vast)
```

```suggestion
set(CPACK_DEBIAN_COMPRESSION_TYPE "gzip")
set(CPACK_DEBIAN_FILE_NAME "DEB-DEFAULT")
set(CPACK_DEBIAN_PACKAGE_NAME "vast")
```

For consistency with the rest of our CMake.